__tests__/validateAuth.tsx

```diff
@@ -0,0 +1,35 @@
+import {getBooks} from 'src/services/booksService';
+import api from 'src/config/api';
+import {validateEmail, validatePassword} from 'src/utils/validations';
+
+describe('validate Password', () => {
+  test('invalid password', () => {
+    expect(validatePassword('asd')).toBe('Contraseña invalida');
+  });
+  test('valid password', () => {
+    expect(validatePassword('wowowo1134')).toBe(undefined);
+  });
+  test('required password', () => {
+    expect(validatePassword('')).toBe('Contraseña requerida');
+  });
+});
+
+describe('validate email', () => {
+  test('invalid password', () => {
+    expect(validateEmail('asd')).toBe('Email invalido');
+  });
+  test('valid password', () => {
+    expect(validateEmail('ignacio.coluccio@wolox.com.ar')).toBe(undefined);
+  });
+  test('required password', () => {
+    expect(validateEmail('')).toBe('Email requerido');
+  });
+});
+
+describe('books services', () => {
+  api.get = jest.fn().mockReturnValue({});
+  test('getBooks', () => {
+    const response = getBooks();
+    expect(response).toEqual({});
+  });
```

Comment:
What are you testing here? You should mock answers and check your fluxes. It doesn't have any sense test a service call only. You should mock the response of this service and check the differents flows in your app if it has sense. For example if you have different injectios or flows attached to this service call, you should mock the service call with the different cases (Failure, Success, ERROR_CODES if you do some logig based on it, etc).